.travis.yml

```diff
@@ -1,19 +1,25 @@
 ---
 language: node_js
 node_js:
-  - "4"
+  - 6
```

Comment:
We'd like to maintain Node 4 compatibility, especially because there are Node 4 LTS releases that don't fall out of maintenance until April 2018 (https://github.com/nodejs/Release).

Can you add back Node 4 here, and see comment below in package.json about Node 4 version.

Reply:
Yes, no problem. I'll add that back in.

I initially removed it because I was able to have the add-on build and pass all the tests locally running Node 4, however Travis was throwing errors. The error seemed to be related to a missing "use strict;", but I had added it to the referenced file. So I think there may be a caching or other issue.